sources/platform/actors/development/actor_definition/dataset_schema/validation.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Dataset validation
+description:  Specify the dataset schema within the Actors so you can add monitoring and validation down to the field level.
+slug: /actors/development/actor-definition/dataset-schema/validation
+---
+
+**Specify the dataset schema within the Actors so you can add monitoring and validation down to the field level.**
+
+---
+
+To define a schema for a default dataset of an Actor run, you need to set `fields` property in the dataset schema. It’s currently impossible to set a schema for a named dataset (same as for dataset views).
```

Comment:
```suggestion
To define a schema for a default dataset of an Actor run, you need to set `fields` property in the dataset schema. Apify does not currently support schema for named datasets (same as for dataset views).
```

Reply:
> It’s currently impossible

just feels to me kind of strong, no strong opinion here

Reply:
I would also drop `currently`. I am not a fan of this phrasing within docs, since reader has no idea what is `currently` unless they dive head first into commit history of docs repo. 

I would suggest something like

```suggestion
To define a schema for a default dataset of an Actor run, you need to set `fields` property in the dataset schema.
```

or maybe with an admonition if we want to keep this information (my personal favourite), once we support this we can just delete the admonition

```suggestion
To define a schema for a default dataset of an Actor run, you need to set `fields` property in the dataset schema.

:::note Schema limitations

Schema configuration is not available for named datasets or dataset views.

:::
```